MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -13,17 +13,39 @@ final class MovieQuizViewController: UIViewController {
     private var currentQuestion: QuizQuestion?
     private var alertPresenter: AlertPresenterProtocol?
     private var statisticService: StatisticServiceProtocol?
+    private var moviesLoader: MoviesLoadingProtocol?
 
     @IBOutlet private weak var counterLabel: UILabel!
     @IBOutlet private weak var imageView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var noButton: UIButton!
     @IBOutlet private weak var yesButton: UIButton!
+    @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
     
     // MARK: - Controller class methods
+    private func showLoadingIndicator() {
+        activityIndicator.isHidden = false
+        activityIndicator.startAnimating()
+    }
+    
+    private func showNetworkError(message: String) {
+        activityIndicator.isHidden = true
+        let alertModel = AlertModel(
+            title: "Ошибка",
+            message: message,
+            buttonText: "Попробовать ещё раз",
+            completion: { [weak self] in
+                guard let self = self else { return }
+                self.currentQuestionIndex = 0
+                self.correctAnswers = 0
+                self.questionFactory?.loadData()
```

Comment:
Очень хорошо такой код выносить в отдельный метод! тогда в замыканиях нам становится чуточку проще работать, 
```suggestion
                self?.resetGame()
```
Во первых одной строчкой без разыменовывания self можно решить необходимую задачу. Во вторых, это помогает читаемости кода: разработчики всегда стремятся писать более плоский код(без глубоких табов). 
Чем короче вложенный код – тем лучше. 
Ну и в третьих – мы работаем над гибкостью и переиспользуемостью: возможно этот же кусочек повторяется при окончании игры, и его можно перевызвать. (а может нет, если там не нужно запрашивать вопросы заново)